File: src/main/Java/boj_step3/boj10952.java
import java.io.*;
import java.util.StringTokenizer;

public class boj10952 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        while (true) {
            String string = reader.readLine();
            if (string.equals("0 0")) { writer.close(); return; }
            StringTokenizer tokenizer = new StringTokenizer(string);
            int a = Integer.parseInt(tokenizer.nextToken());
            int b = Integer.parseInt(tokenizer.nextToken());
            writer.append(String.valueOf(a + b));
            writer.newLine();
        }
    }
}

File: src/main/kotlin/others/boj10952.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    while (true) {
        val (a, b) = r.readLine()!!.split(" ").map { it.toInt() }
        if (a == 0 && b == 0) break
        w.write((a + b).toString() + "\n")
    }
    w.close()
}